### PR TITLE
Added set_max_velocity_scaling_factor function to move_group.py

### PR DESCRIPTION
**Pull request by ferherranz**

### CODE REVIEW

File: src/moveit_commander/move_group.py [hunk 1]
```diff
@@ -389,6 +389,13 @@ def set_workspace(self, ws):
                 else:
                     raise MoveItCommanderException("Expected 0, 4 or 6 values in list specifying workspace")
 
+    def set_max_velocity_scaling_factor(self, value):
+        """ Set a scaling factor for optionally reducing the maximum joint velocity. Allowed values are in (0,1]. """        
+        if value > 0 and value <= 1:
+            self._g.set_max_velocity_scaling_factor(value)
+        else:
+            raise MoveItCommanderException("Expexted value in the range from 0 to 1 for scaling factor" )
```
Comment (davetcoleman) [inline src/moveit_commander/move_group.py]: misspelling "Expexted"

**Comment by davetcoleman** [comment]
+1 except spelling

**Comment by achim-k** [comment]
@ferherranz Could you fix this small typo? Thanks

**Comment by ferherranz** [comment]
Solved!

**Comment by xiangzhi** [comment]
Hi, Is there any chance this pull request will be accepted? It will be helpful if this method is in the official interface.

**Comment by davetcoleman** [comment]
cherry-picked to jade-devel also

**Pull request merged**